ros2_batch_job/__main__.py

```diff
@@ -130,6 +130,10 @@ def main(sysargv=None):
                 'tlsf',
                 'tlsf_cpp',
             ]
+        # There are no debug packages available for PyQt5 and PySide, so we
```

Comment:
Why should this be conditional of the packaging flag? Shouldn't it be one level less indented?

Reply:
Fair point.  I was trying to keep it as restricted as possible, but I guess if we ever turn on "Debug" for packaging, it could hit there as well.  I'll move it out one level.